website/docs/r/data_factory_linked_service.html.markdown

```diff
@@ -0,0 +1,102 @@
+---
+subcategory: "Data Factory"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_data_factory_linked_service"
+description: |-
+  Manages a Linked Service (connection) between a resource and Azure Data Factory.
+---
+
+# azurerm_data_factory_linked_service
+
+Manages a Linked Service (connection) between a resource and Azure Data Factory.
```

Comment:
and here - and add a note this can be used for services without a first part resrouce